release/models/network-instance/openconfig-evpn.yang

```diff
@@ -645,12 +674,27 @@ module openconfig-evpn {
       Using Ethernet VPN";
 
     leaf vni {
+      when "../../../config/service-type = 'oc-evpn-types:VLAN_BASED' or
+            ../../../config/service-type = 'oc-evpn-types:VLAN_BUNDLE'" {
+        description
+          "For VLAN-based and VLAN-bundle EVIs, use a single VNI";
+      }
```

Comment:
Add reference "RFC 7432: BGP MPLS-Based Ethernet VPN section-6";

Reply:
Likewise, one of the previous PRs that has already merged